packages/vscode-extension/src/controls/webviewPanel.ts

```diff
@@ -173,12 +173,7 @@ export class WebviewPanel {
     if (res.isOk()) {
       props[TelemetryProperty.Success] = TelemetrySuccess.Yes;
       ExtTelemetry.sendTelemetryEvent(TelemetryEvent.DownloadSample, props);
-      await globalStateUpdate(GlobalKey.OpenSampleReadMe, true);
```

Comment:
Will GlobalKey.OpenSampleReadMe updated after this change?

Reply:
Yes, it's updated in handlers.ts:L518